web/src/main/java/org/fao/geonet/Geonetwork.java

```diff
@@ -475,6 +477,16 @@ public Object start(Element config, ServiceContext context) throws Exception {
             Integer dbHeartBeatFixedDelay = Integer.parseInt(handlerConfig.getValue(Geonet.Config.DB_HEARTBEAT_FIXEDDELAYSECONDS, "60"));
             createDBHeartBeat(context.getResourceManager(), gnContext, dbHeartBeatInitialDelay, dbHeartBeatFixedDelay);
         }
+
+        boolean linkMonitorEnabled = Boolean.parseBoolean(handlerConfig.getValue(Geonet.Config.LINK_MONITOR_ENABLED, "false"));
```

Comment:
This is actually the code I'm planning to try and push upstream. It's pretty difficult to try changing all of that, so I conformed with what was there before (as you said, `DBHeartBeat`).